ui/src/TimeOnProductPage/TimeOnProduct.tsx

```diff
@@ -110,7 +104,7 @@ function TimeOnProduct({currentSpace}: TimeOnProductProps): JSX.Element {
     }, [products]);
 
     const onNameClick = (timeOnProductItem: TimeOnProductItem): void => {
-        const product = products.find(item => timeOnProductItem.productName === item.name);
+        const product = products.find(item => timeOnProductItem.productName.toLowerCase() === item.name.toLowerCase());
```

Comment:
This fixed the bug on the "Time On Product" page where you couldn't click on the peoples names that weren't assigned a product. The bug was caused because one of these variables was uppercase and one was lowercase.